src/components/AccessReviewHeader.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { div, a, hh } from "react-hyperscript-helpers";
+import { Theme } from '../libs/theme';
+import { Storage } from "../libs/storage";
+import { Navigation } from "../libs/utils";
+
+const TITLE = {
+  fontWeight: Theme.font.weight.semibold,
+  fontSize: Theme.font.size.title,
+  lineHeight: Theme.font.leading.title,
+  marginBottom: '5px',
+}
+
+const SMALL = {
+  fontSize: Theme.font.size.small,
+  lineHeight: Theme.font.leading.dense,
+}
+
+const BUTTON = {
+  ...SMALL,
+  fontWeight: Theme.font.weight.semibold,
+}
+
+export const AccessReviewHeader = hh(class AccessReviewHeader extends React.PureComponent {
+
+  render() {
+    const currentUser = Storage.getCurrentUser();
+    const { history } = this.props;
+    return div(
+      {
+        style: {
+          justifyContent: "space-between",
+          alignItems: "center",
+          display: "flex",
```

Comment:
nit: try running eslint ... I think it will change these double quotes to single quotes.